uiowa/uiowa
# Feature: admissions menu hotfix

Resolves https://github.com/uiowa/uiowa/issues/5375. 

# How to test

- `blt frontend`
- Admissions dev testing `ddev blt sync --site admissions.uiowa.edu --define drush.aliases.remote=admissions.dev`
- Admissions testing: `ddev blt sync --site=admissions.uiowa.edu`
- Tippie testing: `ddev blt sync --site=tippie.uiowa.edu`

## Test on admissions dev dropdowns:

https://admissions.dev.drupal.uiowa.edu/first-year-visit-options
- Verify that this page has active menu styles (due to the unique way this menu is set up, the gold divider lines still appear next to the active menu item.  On menus that are set up normally the divider line next to the active menu items are removed).

<img width="602" alt="Screen Shot 2022-06-17 at 12 04 52 PM" src="https://user-images.githubusercontent.com/1036433/174345167-618e40b6-4225-4d6c-83c8-f70bfdf59fbd.png">

- Go to child pages of this dropdown and verify that parent page retains active button styling

<img width="275" alt="Screen Shot 2022-06-17 at 9 53 02 AM" src="https://user-images.githubusercontent.com/1036433/174322744-a59cc9bd-f7ea-473a-a906-a9b46a7e0c0a.png">


##  Test on admissions local:

Verify that these pages have active menu style:

- https://admissions.uiowa.ddev.site/finances/estimated-costs-attendance
- https://admissions.uiowa.ddev.site/first-year-visit-options

<img width="934" alt="Screen Shot 2022-06-17 at 12 06 30 PM" src="https://user-images.githubusercontent.com/1036433/174345359-f979969c-8ddc-43dc-a539-f24b4c3af274.png">


## Test on Tippie and verify that all menu styles are still working:
- Click through https://tippie.uiowa.ddev.site/iowa-mba and check styles at different menu depths.
